docs/Toy/toy/interpreter.py

```diff
@@ -98,11 +107,12 @@ def run_mul(
 
         return (Tensor([l * r for l, r in zip(lhs.data, rhs.data)], lhs.shape),)
 
-    @impl(toy.ReturnOp)
+    @impl_terminator(toy.ReturnOp)
     def run_return(
         self, interpreter: Interpreter, op: toy.ReturnOp, args: tuple[Any, ...]
-    ) -> tuple[Any, ...]:
-        return args
+    ) -> tuple[TerminatorValue, PythonValues]:
```

Comment:
What is the use of `PythonValues` here?
Is it in the case where the terminator has return values (which is cursed, but allowed)?

Reply:
Exactly, for spooky uses that I can't think of, but technically could happen